src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const element = document.createElement('div');
+  const body = document.querySelector('body');
+  const p = document.createElement('p');
+  const h2 = document.createElement('h2');
+
+  element.classList.add('notification', type);
+  h2.classList.add('title');
+
+  body.prepend(element);
+
+  element.style.top = posTop + 'px';
+  element.style.right = posRight + 'px';
+  p.textContent = description;
+  h2.textContent = title;
+
+  element.append(h2);
+  element.append(p);
+
+  const remove = () => {
+    element.style.display = 'none';
+  };
+
+  window.setTimeout(remove, 2000);
```

Comment:
The timeout for removing the notification is set to 2000 milliseconds (2 seconds). Ensure this duration is appropriate for your use case, as it might be too short for users to read the notification content.